index.html

```diff
@@ -9,47 +9,73 @@
+                    
+                        <select name="alcohol" id="alcohol-select">
+                            <option value="">Choose your booze</option>
+                            <option value="bourbon">Bourbon</option>
+                            <option value="brandy">Brandy</option>
+                            <option value="cognac">Cognac</option>
+                            <option value="gin">Gin</option>
+                            <option value="rum">Rum</option>
+                            <option value="scotch">Scotch</option>
+                            <option value="tequila">Tequila</option>
+                            <option value="triplesec">Triple Sec</option>
+                            <option value="vodka">Vodka</option>
+                            <option value="whiskey">Whiskey</option>
+                        </select>
+                        <!--not sure about the toggle, for now made into a "checkbox"-->
+                        <div>
+                            <input type="checkbox" id="non-alcoholic" name="non-alcoholic" />
+                            <label for="non-alcoholic">Non-alcoholic</label>
+                        </div>
+                    
+                        <div class="submit-container">
+                            <input type="submit" value="Tell Me!" />
+                        </div>
+                    </form>
+                </div>
+            </div>
         </header>
         <main>
-            <!--not sure about the toggle, for now made into a "checkbox"-->
-
-            <div>
-                <input
-                    type="checkbox"
-                    id="non-alcoholic"
-                    name="non-alcoholic"
-                />
-                <label for="non-alcoholic">Non-alcoholic</label>
-            </div>
-
-            <div class="submit-container">
-                <input type="submit" value="Tell Me!" />
-            </div>
-
             <section class="results">
-                <!-- dynamic results will be added here -->
+                <h2><!-- dynamic results will be added here --></h2>
+                <!-- .buttonContainer will be hidden when .reveal button is pressed: -->
+                <div class="buttonContainer">
+                    <button class="reveal">Let's Make It!</button>
+                    <button class="randomize">That's Not It...</button>
+                </div>               
+            </section>
+            <!-- .recipe section will be revealed when .reveal button is pressed: -->
+            <section class="recipe">
+                <div class="drinkImage">
+                    <!-- strImageSource will be appended here (.appendChild) -->
+                </div>
+                <!-- .recipeContainer div will just be for styling purposes: -->
+                <div class="recipeContainer">
+                    <div class="ingredients">
+                        <h4>INGREDIENTS</h4>
```

Comment:
Should we use `text-transform` instead of caps in the HTML?

Reply:
Yes. Absolutely.